ui/lesson/ScriptingChallenge/Runner/index.tsx

```diff
@@ -85,15 +85,28 @@ export default function Runner({
 
         switch (action) {
           case 'log': {
-            print(payload)
+            typeof payload === 'string' && print(payload)
             break
           }
           case 'result': {
-            setResult(payload)
+            typeof payload.value === 'string' && setResult(payload)
 
             if (payload.error) {
-              setErrors([payload.error])
-              setHasherState(HasherState.Error)
+              if (payload.error.split('Traceback')[1]) {
```

Comment:
This will become messy quickly, lets instead add language-specific error handling to their respective files i.e. `ui\lesson\ScriptingChallenge\compilers\python.ts`. It currently exports a `compile` function, lets also have it export a `handleError` function or something similar and call the correct one based on the `language` that is passed via props to this component, rather than attempting to split on 'Traceback', which could break when we add more languages.